docs/02-app/01-building-your-application/04-caching/03-data-cache.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: Data Cache
+description: Learn more about persistent data caching in Next.js
+---
+
+Next.js has a built-in data caching mechanism that stores the result of data requests across user requests and deployments. This lowers cost and improves performance by reducing the number of requests to your data source.
+
+## How it works
+
+Next.js extends the native [Web fetch() API]() to allow each request on the server to set its own persistent caching semantics. By default, data requests that use `fetch` are automatically cached, and you can use the [`cache`]() and [`next.revalidate`]() options of `fetch` to configure the caching behavior.
+
+When fetching data on the server, a `fetch` request first checks the persistent HTTP cache for a cached response. If a cached response is found, it is returned immediately. If not, the request is made to the data source, and the response is cached.
+
+> **Good to know:**
+>
+> In the browser, the `cache` option indicates how a fetch request will interact with the browser's HTTP cache. By extending the `fetch` API, you can use `cache` to indicate how a server-side fetch request will interact with the framework's persistent HTTP cache.
+
+## Duration and Invalidation
+
+The cache is persistent across user requests and deployments, unless you revalidate or opt out of caching.
+
+### Revalidation
+
+Revalidation is the process of purging the cache and re-fetching the latest data. This is useful when your data changes and you want to ensure you show the latest information.
+
+Cached data can be revalidated in two ways:
+
+- **Time-based revalidation**: Automatically revalidate data after a certain amount of time has passed. This is useful for data that changes infrequently and freshness is not as critical. Time-based revalidation can be configured with the [Route Segment Config options]() or per invididual [`fetch` request]().
```

Comment:
Since it is called "[Background Revalidation](https://nextjs.org/docs/app/building-your-application/data-fetching/revalidating#background-revalidation)" here, how about unifying it?

Reply:
Agree, after we've penned down the explanation. I'll go through the docs, consolidate terms, and update stale information.